src/traverseAndModifyAst.ts

```diff
@@ -15,6 +15,14 @@ export function traverseAndModifyAst(ast: Node) {
 			modifyNodeIfMissingBrackets(node) &&
 			nodeNotAlreadySeen(node, seenNodes)
 		) {
+			// Remove comments from the original node to avoid duplication
+			if (node.leadingComments) {
+				delete node.leadingComments;
+			}
+			if (node.trailingComments) {
+				delete node.trailingComments;
+			}
```

Comment:
[Question] Any reason for the `if`s? No tests fail if I simplify them out:

```suggestion
			delete node.leadingComments;
			delete node.trailingComments;
```

Reply:
Good catch :D I forgot delete doesn't throw if the property doesn't exist on the object, so the if statements are unnecessary. I've removed them